### PR TITLE
Funder fees

### PR DESCRIPTION
**Pull request by niftynei**
Not accounting for our fees when we're calculating the total available for funding bit @fiatjaf 

If a utxo isn't economical, we shouldn't add it to the transaction.

### CODE REVIEW

File: plugins/funder.c [hunk 1]
```diff
@@ -219,16 +220,20 @@ json_openchannel2_sign_call(struct command *cmd,
...
+		if (psbt_input_is_ours(&psbt->inputs[i])) {
 			json_add_num(req->js, NULL, i);
+			count++;
+		}
 	}
 	json_array_end(req->js);
 
 	plugin_log(cmd->plugin, LOG_DBG,
-		   "calling `signpsbt` for channel %s",
+		   "calling `signpsbt` for channel %s for %zu input%s",
```
Comment (vincenzopalazzo) [inline plugins/funder.c:233]: Maybe there is missing space in `input%s`?

**Review by vincenzopalazzo — CHANGES_REQUESTED** [review]
ack https://github.com/ElementsProject/lightning/commit/d2ab0b14b15cd3566acdd0fbbda967e97d30f927 with only a space missing I think

File: plugins/funder.c (hunk 1 above)
Comment (niftynei) [reply plugins/funder.c:233]: The last `%s` is for gramamtical reasons. It'll print "1 input" or "2 inputs".

Comment (vincenzopalazzo) [reply plugins/funder.c:233]: My big mistake, sorry I only take into consideration the line and not the meaning! (bad job vincent :) ).

**Review by vincenzopalazzo — APPROVED** [review]

**Comment by rustyrussell** [comment]
Ack 1d1d64ea6a6caf87db0deaa4cec64d17a80f5e01

**Pull request merged**